cc-pk-service/test/pk.test.ts

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+const mock = new MockAdapter(axios);
```

Comment:
I had problems when making multiple tests with MockAdapter and Axios. It seems, that we should create an instance of axios for all tests (at least in one service) to use and also use MockAdapter.reset() and MockAdapter.restore(). Not yet sure how these should be implemented so I just put all the current route unit tests in same file. Then there are no axios problems. 

But nevertheless, this PR should be merged and the problem mentioned should be addressed in another PR.